packages/orbit-components/src/utils/Grid/Grid.stories.tsx

```diff
@@ -1,100 +1,75 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import { SPACINGS_AFTER } from "../../common/consts";
 
 import Grid from ".";
 
-export default {
-  title: "Grid",
-};
+type GridPropsAndCustomArgs = React.ComponentProps<typeof Grid> & { divsCount: number };
 
-export const Playground = ({
-  inline,
-  maxWidth,
-  width,
-  columns,
-  rows,
-  gap,
-  columnGap,
-  rowGap,
-  as,
-  dataTest,
-  divsCount,
-  mediumMobile,
-  largeMobile,
-  tablet,
-  desktop,
-  largeDesktop,
-  spaceAfter,
-}) => {
-  return (
-    <Grid
-      inline={inline}
-      spaceAfter={spaceAfter}
-      maxWidth={maxWidth}
-      width={width}
-      columns={columns}
-      rows={rows}
-      gap={gap}
-      columnGap={columnGap}
-      rowGap={rowGap}
-      as={as}
-      dataTest={dataTest}
-      mediumMobile={mediumMobile}
-      largeMobile={largeMobile}
-      tablet={tablet}
-      desktop={desktop}
-      largeDesktop={largeDesktop}
-    >
-      {Array(...Array(divsCount)).map((_, key) => (
-        // eslint-disable-next-line
-        <div key={key} style={{ background: "rgba(0, 169, 145, 0.2)" }} />
-      ))}
-    </Grid>
-  );
-};
+const meta: Meta<GridPropsAndCustomArgs> = {
+  title: "Grid",
+  component: Grid,
 
-Playground.args = {
-  inline: false,
-  maxWidth: "1440px",
-  width: "100%",
-  columns: "",
-  rows: "repeat(8, 40px)",
-  gap: "",
-  columnGap: "",
-  rowGap: "10px",
-  as: "div",
-  dataTest: "test",
-  divsCount: 8,
-  mediumMobile: {
-    rowGap: "0",
-  },
-  largeMobile: {
-    columns: "repeat(4, 1fr)",
-    rows: "repeat(2, 40px)",
-    gap: "20px",
-  },
-  tablet: {
-    columnGap: "40px",
-  },
-  desktop: {
-    columns: "repeat(2, minmax(100px, 1fr))",
-    rows: "repeat(4, 40px)",
-    gap: "40px",
+  args: {
+    divsCount: 8,
```

Comment:
`divsCount` are not visible in case it's higher then number of rows on `row`. I understand that it can be tricky, in that way I would add a note to `info` and inform the user. WDYT?

Reply:
I added more info about this, I hope it's sufficient like this :)